src/chain-data-utils/combined_queries.ts

```diff
@@ -3,7 +3,7 @@ import { clusterApiUrl, Connection } from "@solana/web3.js";
 import {
   latestAccountInputsOnDateSolanaFm,
   tokenAccountsInfoBetweenDatesSolanaFm,
-} from "./solanaFm";
+} from "./solana_fm";
```

Comment:
Haha thanks, got really sloppy with the file names with no linter to keep me in check 😞  I think I have some mixed camel and snake case in the code too, especially since I tried keeping the db columns as snake case

Reply:
Yeah I actually didn't mind the DB names being snake case. That's a pattern I'll use as well for external JSON data I'm interacting with or generally IO data you don't control. Sort of a nice quick visual to know whether data is all TS/JS or from some other source.